hawkular-alerts-actions/hawkular-alerts-actions-plugins/hawkular-alerts-actions-webhook-global/hawkular-alerts-actions-webhook-global-plugin/pom.xml

```diff
@@ -22,13 +22,13 @@
 
   <parent>
     <groupId>org.hawkular.alerts</groupId>
-    <artifactId>hawkular-alerts-actions-plugins</artifactId>
+    <artifactId>hawkular-alerts-actions-webhook-global</artifactId>
     <version>1.1.5.Final-SNAPSHOT</version>
   </parent>
 
-  <artifactId>hawkular-alerts-actions-snmp</artifactId>
-  <packaging>war</packaging>
+  <artifactId>hawkular-alerts-actions-webhook-global-plugin</artifactId>
```

Comment:
Yes, I guess I copied the pom.xml used for snmp as base to customize the plugins.
Anyway, I will review them, just in case some of them are missing, I noticed in a previous comment that an artifact name was not properly set up.
It should be an easy fix.